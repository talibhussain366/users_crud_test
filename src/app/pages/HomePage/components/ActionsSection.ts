import styled from 'styled-components/macro';

export const ActionsSection = styled.div`
  display: flex;
  justify-content: flex-end;
  font-size: 12px;
`;
