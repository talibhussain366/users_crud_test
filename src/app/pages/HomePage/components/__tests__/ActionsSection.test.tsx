import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { ActionsSection } from '../ActionsSection';

const shallowRenderer = createRenderer();

describe('<ActionsSection />', () => {
    it('<ActionsSection />', () => {
        shallowRenderer.render(<ActionsSection />);
        const renderedOutput = shallowRenderer.getRenderOutput();
        expect(renderedOutput).toMatchSnapshot();
    });
});
