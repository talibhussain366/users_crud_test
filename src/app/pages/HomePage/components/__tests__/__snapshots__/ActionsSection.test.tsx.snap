// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionsSection /> <ActionsSection /> 1`] = `
<div
  className="ActionsSection-sc-3cmt0s-0 lbvzQo"
/>
`;
